hr_payroll_expense/data/hr.salary.rule.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+File: hr.salary.rule.xml
+Module: hr_payroll_expense
+
+Created by dco@open-net.ch
+
+Copyright (c) 2014 Open-Net Ltd.
+-->
```

Comment:
@elicoidal @dreispt I've been asking myself this question: Is it accepted by the community to put copyrights on data and views ? In my opinion, it seams a bit awkward.